exercises/concept/savings-account/savings_account_test.rb

```diff
@@ -81,7 +81,7 @@ def test_annual_balance_update_for_small_negative_start_balance
   end
 
   def test_annual_balance_update_for_average_negative_start_balance
-    assert_in_delta 1_016.21, SavingsAccount.annual_balance_update(1_000.0), 0.01
+    assert_in_delta -1_032.13, SavingsAccount.annual_balance_update(-1_000.0), 0.01
```

Comment:
```suggestion
    assert_in_delta -1_032.13, SavingsAccount.annual_balance_update(-1_000.0), 0.01 # rubocop:disable Lint/AmbiguousOperator
```

This is how we might state explicitly that the syntax is what we prefer specifically here, even if generally we agree with using parenthesis around negative numbers.

Otherwise, we should use

```suggestion
    assert_in_delta(-1_032.13), SavingsAccount.annual_balance_update(-1_000.0), 0.01
```